application/modules/apiV2/controllers/WorkflowController.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+require_once 'V2BaseController.php';
+
+class ApiV2_WorkflowController extends V2BaseController
+{
+
+    /**
+     * @OA\Post(
+     *     path="/workflow",
+     *     tags={"workflow"},
+     *     summary="Execute workflow",
+     *     description="Execute a workflow",
+     *     operationId="execute",
+     *     @OA\Parameter(
+     *         name="execute",
+     *         in="query",
+     *         description="name of workflow",
+     *         required=true,
+     *         @OA\Schema(
+     *             type="string",
+     *         )
+     *     ),
+     *     @OA\RequestBody(
+     *          required=false,
+     *          @OA\MediaType(
+     *              mediaType="application/json",
+     *              @OA\Schema(
+     *                  type="object",
+     *                  required={},
+     *                  @OA\Property(
+     *                      type="object",
+     *                      property="workflow",
+     *                      description="workflow name",
+     *                      @OA\Property(
+     *                          type="object",
+     *                          property="params",
+     *                          description="workflow params",
+     *                              example={ "ci_id": 12 }
+     *                      ),
+     *                      @OA\Property(
+     *                          type="string",
+     *                          property="startAt",
+     *                          description="start at ISO8601",
+ *                              example="2004-02-12T15:19:21+00:00"
+     *                      ),
+     *                      @OA\Property(
+     *                          type="bool",
+     *                          property="forceAsync",
+     *                          description="start the workflow always async. _false_ can't override async workflows to be run sync",
+ *                              example=false
+     *                      ),
+     *                  ),
+     *              ),
+     *          ),
+     *     ),
+     *     @OA\Response(
+     *         response=200,
+     *          description="CI saved successfully",
```

Comment:
Please fix the description.
Async workflow executions should return 202 Accepted instead.